<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>üîç Searching Counsellors</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	
	border-image: url(GUI_Elements/bg image.png);
}
QLabel {
  color: #464d55;
  font-weight: 600;
}
QLabel#heading {
  color: #0f1925;
  font-size: 18px;
  margin-bottom: 10px;
}

QLabel#subheading {
  color: #0f1925;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}
QLineEdit {
  background: transparent;
  color: rgb(86, 86, 86);
  border-bottom: 1px solid rgb(86, 86, 86);
}

QPushButton {
  background-color: #85E3D9;
  color: #0f1925;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #85E3D9;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover,
QPushButton:focus {
  background-color: #85E3D9;
  border: 3px solid #85E3D9;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;

color: rgb(86, 86, 86);</string>
     </property>
     <property name="text">
      <string>Filter By</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="country">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string> Country Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: rgb(86, 86, 86);</string>
     </property>
     <property name="text">
      <string>Country</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
    </property>
    <property name="placeholderText">
     <string> üîç Search Counsellors</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="counsellorWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;</string>
    </property>
    <column>
     <property name="text">
      <string>Counsellor Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contact Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contact Number</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="searhButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover,
QPushButton:focus {
	
  color: rgb(255, 255, 255);
  background-color: #85E3D9;
  border: 3px solid #85E3D9;
}</string>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backbutton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover,
QPushButton:focus {
	
  color: rgb(255, 255, 255);
  background-color: #85E3D9;
  border: 3px solid #85E3D9;
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="GUI Elements/SearchGradient.qrc"/>
 </resources>
 <connections/>
</ui>
